src/Stratis.Bitcoin/P2P/PeerConnector.cs

```diff
@@ -75,7 +75,7 @@ public abstract class PeerConnector : IPeerConnector
         public int MaxOutboundConnections { get; set; }
 
         /// <summary>Global application life cycle control - triggers when application shuts down.</summary>
-        protected INodeLifetime nodeLifetime;
+        protected INodeLifetime NodeLifetime { get; private set; }
```

Comment:
Same as above

Reply:
This is right no? I changed this to a property (it is not a field anymore).